src/utils/pause/mapScopes/findGeneratedBindingFromPosition.js

```diff
@@ -95,15 +135,43 @@ async function findGeneratedReference(
     end: Location
   }
 ): Promise<GeneratedDescriptor | null> {
-  return generatedAstBindings.reduce(async (acc, val) => {
+  const bindings = filterApplicableBindings(generatedAstBindings, mapped);
+
+  return bindings.reduce(async (acc, val) => {
+    const accVal = await acc;
+    if (accVal) {
+      return accVal;
+    }
+
+    return mapBindingReferenceToDescriptor(val, mapped);
+  }, null);
+}
+
+async function findGeneratedImportReference(
+  type: BindingType,
+  generatedAstBindings: Array<GeneratedBindingLocation>,
+  mapped: {
+    type: BindingLocationType,
+    start: Location,
+    end: Location
+  }
+): Promise<GeneratedDescriptor | null> {
+  let bindings = filterApplicableBindings(generatedAstBindings, mapped);
+
+  // When wrapped, for instance as `Object(ns.default)`, the `Object` binding
+  // will be the first in the list. To avoid resolving `Object` as the
+  // value of the import itself, we potentially skip the first binding.
+  if (bindings.length > 1 && !bindings[0].meta && bindings[1].meta) {
```

Comment:
This was working by change before because of the bad sorting. Now that I fixed that typo and we're properly comparing `a` and `b` it showed up.